src/helpers.js

```diff
@@ -69,7 +98,7 @@ export const createConnectedIcon = (connected) => {
       }
     >
       <Flex>
-        <DisconnectedIcon className="pf-u-mr-xs" />
+        <DisconnectedIcon className={xsIcon} />
```

Comment:
The same thing: let's wrap it with `Icon` and use the `size` prop out of the box.

Reply:
Done